### PR TITLE
catana: Added catana tool

### PR DESCRIPTION
**Pull request by D3vil0per**
Added PKGBUILD file for catana tool (source: https://github.com/D3vil0per/catana)

### CODE REVIEW

File: packages/catana/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
```
Comment (noraj) [inline packages/catana/PKGBUILD:5]: This not the version returned by `pkgver()`, checkout our templates. https://github.com/BlackArch/blackarch-pkgbuilds

File: packages/catana/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+pkgrel=1
+pkgdesc='CATANA filters your wordlist according to the specified password policy.'
```
Comment (noraj) [inline packages/catana/PKGBUILD:7]: Can you remove tool name from the description? it's redundant

File: packages/catana/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+groups=('blackarch' 'blackarch-misc')
+arch=('any')
+url='https://github.com/D3vil0per/catana'
+license=('GNU General Public License v3.0')
```
Comment (noraj) [inline packages/catana/PKGBUILD:11]: It's not how it works. https://wiki.archlinux.org/title/PKGBUILD#license

File: packages/catana/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+depends=('bash')
+makedepends=('git')
+source=("git+https://github.com/D3vil0per/$pkgname.git")
+
+pkgver() {
+	cd $pkgname
```
Comment (noraj) [inline packages/catana/PKGBUILD:17]: you have mixed indentation.

File: packages/catana/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+	cd $pkgname
+
+  install -Dm 755 -t "$pkgdir/usr/bin/" catana
+}
+sha256sums=('SKIP')
```
Comment (noraj) [inline packages/catana/PKGBUILD:27]: could you put it under source please and we use `sha512sums`

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/catana/PKGBUILD (hunk 2 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Deleting application name from description.

**Review by D3vil0per — COMMENTED** [review]
Managed all the requests.

File: packages/catana/PKGBUILD (hunk 1 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Changed in commit 550e72f80ebc3d7322f2ceed2d2f737ad3494375

File: packages/catana/PKGBUILD (hunk 2 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Commit: 49865587c417d0345a817706d2cab8f25cc7506f

File: packages/catana/PKGBUILD (hunk 3 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Commit a9867a8b3d55ce47cf7cf344bffc2838f7856422

File: packages/catana/PKGBUILD (hunk 4 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Commit 9090a8bfaf716fabb0923bcf318e2de25821a587

File: packages/catana/PKGBUILD (hunk 5 above)
Comment (D3vil0per) [reply packages/catana/PKGBUILD]: Commit 4eb29ca65934a806b201d6c3e6e452b3b8fc0640

**Comment by D3vil0per** [comment]
@noraj managed all the requests you asked for. I commented each conversation with commit number. Is there another step on my side for merging the new content?

**Comment by noptrix** [comment]
thanks guys

**Pull request merged**